rita_client/src/dashboard/extender_checkin.rs

```diff
@@ -0,0 +1,40 @@
+use actix_web_async::{web::Json, HttpResponse};
+use std::{
+    collections::HashMap,
+    sync::{Arc, RwLock},
+};
+
+use rita_common::dashboard::wifi::{
+    get_device_mac, get_wifi_config_internal, ExtenderAdditionalSettings, ExtenderCheckin,
+    ExtenderUpdate,
+};
+
+lazy_static! {
+    /// This is a mapping of Mac Addr of extender -> Its checkin struct
+    /// Do we need to store all this information here? What this rita client need to keep track of?
+    static ref EXTENDER_LIST: Arc<RwLock<HashMap<String, ExtenderCheckin>>> = Arc::new(RwLock::new(HashMap::new()));
+}
+
+pub async fn extender_checkin_handler(extender: Json<ExtenderCheckin>) -> HttpResponse {
+    let extender = extender.into_inner();
+    // Store this in local database, what info do we need to store here
+    add_wifi_extender(extender);
```

Comment:
we need to keep track of what extenders we have connected and also their wifi info so we can upload it. (maybe other hardware info too? we mostly care about wifi info though)

Reply:
doc comments would be good here as well.